cypress/support/fragments/inventory/inventoryActions.js

```diff
@@ -19,4 +19,15 @@ export default {
       cy.expect(element.is({ disabled: false }));
     });
   },
+  verifySaveUUIDsFileName(actualName) {
```

Comment:
"The time stamp is local time of the user running the test" - can't see this check